test/unit/compiler/semantic/IdentificationWalkerTest.scala

```diff
@@ -33,11 +33,8 @@ class IdentificationWalkerTest extends AnyFunSuite with BeforeAndAfter {
   final val logger = Logger[IdentificationWalkerTest]
 
   // In order to be sure that on each test case we do not have data from previous tests.
-  before {
-    logger.debug("Restoring memory symbol table.")
```

Comment:
What happened to the logger?

Reply:
I restored the logging messages in the following commit, my bad.